### DISCUSSION TITLE
What is the nickname of the lead singer of The Police?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#778] [Galaxy Trivia] [answered]
**Bonus:** How did he get this nickname?

**Comment by SANKARAMDAS** [comment]
Sting, byname of Gordon Sumner, original name in full Gordon Matthew Thomas Sumner, (born October 2, 1951, Wallsend, Northumberland, England), British singer and songwriter known both for being the front man of the band the Police and for his successful solo career that followed.

He received the nickname “Sting” from one of his Phoenix Jazzmen bandmates because of the black-and-yellow striped sweater he often wore while performing. After graduating in 1974, Sting taught at St. Paul's First School in Cramlington for two years.

**Comment by peterbaccaro** [comment, marked as answer]
The lead singer of The Police, a British rock band formed in the late 1970s, is **Sting**. Sting's real name is Gordon Matthew Thomas Sumner, but **he adopted his stage name due to the black and yellow striped sweater he used to wear during his early performances**, which reminded one of the band members of a wasp or a bee (hence "Sting").

**The trombone player, Miles Akiyama, gave Sting his nickname during his time with the band Last Exit.**

Sting said: "**He made me sing a song which was awful. So, in protest I began to wear a black and yellow top. He started to call me Sting as a joke. I'm grateful for it now as when you have to sign something, it's short!**".

![image](https://github.com/ParanoidUser/yolo/assets/74666079/09adca40-50bd-4568-939f-2462eaf40d86)
